### PR TITLE
Intercom track

### PR DESCRIPTION
**Pull request by azharmalik3**
@mosic

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -18,11 +18,37 @@ def authenticate_user!
...
       else
         sign_in user
+        update_user_intercom(user)
         redirect_to redirect_url
       end
     end
   end
 
+  def update_user_intercom(user)
+    if Evercam::Config.env == :development
```
Comment (mosic) [inline app/controllers/application_controller.rb]: Why check for development here?

**Comment by mosic** [comment]
Looks mostly good aside from code in the controller, Intercom configuration should be in an initializer.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (azharmalik3) [inline app/controllers/application_controller.rb]: Oh, actually i am testing this on local machine forgot to change.

**Comment by azharmalik3** [comment]
@mosic i am using intercom-ruby gem which is not adding configuration in initializer. I am not using intercom-rails gem because it automatically add intercom to all over the site.

**Comment by mosic** [comment]
@azharmalik3 ok, I just saw the discussion on intercom-ruby repo about that. Looks good :+1:

**Pull request merged**